app/js/components/delete_member.ts

```diff
@@ -11,8 +11,8 @@ $(() => {
       if (!target.matches('.teams__deleteMemberLink')) return;
       event.preventDefault();
 
-      const td = document.querySelector('.teams__actions') as HTMLTableDataCellElement;
-      td.classList.add('loading');
+      const td = $(target).closest('td');
```

Comment:
why are you reverting to jQuery here?  We were phasing out that dependency, are we returning to it?